snippets/facets.liquid

```diff
@@ -157,14 +169,21 @@
                       <fieldset class="facets-wrap parent-wrap {% if filter_type == 'vertical' %} facets-wrap-vertical{% endif %}">
                         <legend class="visually-hidden">{{ filter.label | escape }}</legend>
                         <ul
-                          class="{% if filter_type != 'vertical' %} facets__list{% endif %} list-unstyled no-js-hidden"
+                          class="{{ visual_layout_class }}{% if filter_type == 'vertical' %} facets__list--vertical{% else %} facets__list{% endif %} list-unstyled no-js-hidden"
                           role="list"
                         >
```

Comment:
Since this is `media` I wonder if it should follow the global styles

https://screenshot.click/12-18-7qjvv-eo3le.mp4

Reply:
We talked about it inheriting from global styles, but decided for the first release, to not have it be based on anything except the button radius for thumbnails. @al-mcquiston may be able to better give background, but it was a conscious decision to not attach to global styles.

Reply:
> Have we explored not inheriting at all until we have a better home for such settings?

I am aligned not doing _any_ global settings for this first iteration. I can remove the border radius.

> I would recommend increasing the bottom padding of those thumbnails to improve the white balance.

Updated to 8px, good call out.

> Have we considered reducing the label size as well to help make more room for more letters?

I updated to 1.3rem instead of 1.4rem. It's not a huge difference, but 1.2 looked way too tiny compared to the other filter text.

> Have we explored making the selected state thicker?

We've heard this feedback a few times. @al-mcquiston do you think we should make any changes since it's pretty common feedback?

Reply:
> We've heard this feedback a few times. @al-mcquiston do you think we should make any changes since it's pretty common feedback?

To follow up on this: I've made some changes that Al and I talked about and this has been pushed. We're chatting about the `/` a bit now and might make a couple slight tweaks to the focus, but then we should be good!